README.md

```diff
@@ -9,6 +9,59 @@ The catalog is installed automatically by the Oracle Cloud Native Environment
 CLI.  It is also possible to install the catalog from the catalog itself, by
 installing the `ocne-catalog` chart.
 
+## Support Matrix
+
+### Supported for Oracle Cloud Native Environment 2.0
+
+| Application | Chart | Versions |
+|-------------|-------|----------|
+| Cluster API Core Controller | core-capi | 1.7.1 |
+| Cluster API Kubeadm Control Plane Controller | control-plane-capi | 1.7.1 |
+| Cluster API Kubeadm Bootstrap Controller | bootstrap-capi | 1.7.1 |
+| Cluster API for Oracle Cloud Infrastructure | oci-capi | 0.15.0 |
+| Cert Manager | cert-manager | 1.14.5 |
+| Cert Manager OCI Webhook | cert-manager-webhook-oci | 1.1.0 |
+| Dex | dex | 2.39.1 |
+| ExternalIP Webhook | externalip-webhook | 1.0.0 |
+| Flannel | flannel | 0.22.3 |
+| Fluent Operator | fluent-operator | 3.2.0 |
+| Fluentd | fluentd | 1.14.5 |
+| Grafana | grafana | 7.5.17 |
+| Ingress Nginx | ingress-nginx | 1.9.6 |
+| Istio CRDs | istio-base | 1.20.5 |
```

Comment:
I noticed older versions of istio and rook are missing. is it intentional?

Reply:
Istio 1.19 and earlier versions of Rook are in the upgrade only table.